Feature/upollo sdm 1 (#5697)
segmentio/segment-docs: src/connections/destinations/catalog/upollo/index.md

> ---
> title: Upollo Destination
> ---

Reviewer: ```suggestion
---
title: Upollo Destination
id: 62fc4ed94dd68d0d189dc9b2
beta: true
---
```

[Upollo](https://upollo.ai?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} gives unique and actionable insights that lead to conversion, retention, and expansion.

This destination is maintained by Upollo. For any issues with the destination, [contact Upollo's Support team](mailto:support@upollo.ai).

> info ""
> The Upollo Destination is currently in beta, which means that Upollo still actively developing the destination. If you are interested in joining the beta program or have any feedback to help improve the Upollo Destination and its documentation, [let the Upollo team know](mailto:support@upollo.ai).

## Getting started

1. From the [Upollo Connections screen](https://app.upollo.ai/settings/connections), in the Segment connection, click **Configure**

> 2. Under **I am not a Segment workspace admin**, click **Connect**
> 3. Choose which Source should send data to the Upollo destination.

Reviewer: ```suggestion
2. You will be redirected to Segment to login if you are not already
3. Choose the workspace and source from which you want to send data to Upollo.
4. Click "Authorize," and the destination will be created.
```

Changing the language based off what we display for the OAuth connection: 
<img width="647" alt="Screen Shot 2023-11-22 at 9 49 54 AM" src="https://github.com/segmentio/segment-docs/assets/64277654/e1d26851-ed3b-421a-b17f-d989041f3e23">

Author: Thank you, sounds great.

---
title: Upollo Destination
id: 62fc4ed94dd68d0d189dc9b2
beta: true
---

[Upollo](https://upollo.ai?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} gives unique and actionable insights that lead to conversion, retention, and expansion.

This destination is maintained by Upollo. For any issues with the destination, [contact Upollo's Support team](mailto:support@upollo.ai).

> info ""

> > The Upollo Destination is currently in beta, which means that Upollo still actively developing the destination. If you are interested in joining the beta program or have any feedback to help improve the Upollo Destination and its documentation, [let the Upollo team know](mailto:support@upollo.ai).

Reviewer 2: ```suggestion
> The Upollo Destination is currently in beta, which means that Upollo is still actively developing the destination. If you are interested in joining the beta program or have any feedback to help improve the Upollo Destination and its documentation, [let the Upollo team know](mailto:support@upollo.ai).
```

Author: Hi @forstisabella, thanks for the feedback. Applying it now.

## Getting started

> 1. From the [Upollo Connections screen](https://app.upollo.ai/settings/connections), in the Segment connection, click **Configure**.

Reviewer 2: ```suggestion
1. From the [Upollo Connections screen](https://app.upollo.ai/settings/connections){:target="_blank"}, in the Segment connection, click **Configure**.
```

> 1. From the [Upollo Connections screen](https://app.upollo.ai/settings/connections), in the Segment connection, click **Configure**
> 2. You will be redirected to Segment to login if you are not already

Reviewer: ```suggestion
1. From the [Upollo Connections screen](https://app.upollo.ai/settings/connections), in the Segment connection, click **Configure**.
2. You will be redirected to Segment to login if you are not already.
```

Author: No worries. Thanks.

> 2. You will be redirected to Segment to login if you are not already.

Reviewer 2: ```suggestion
2. Upollo redirects you to Segment. If you're not already logged in to Segment, log in now. 
```

> 3. Choose the workspace and source from which you want to send data to Upollo.

Reviewer 2: ```suggestion
3. Select the workspace and source from which you want to send data to Upollo.
```

> 4. Click "Authorize," and the destination will be created.

Reviewer 2: ```suggestion
4. Click **Authorize**. 
```
